bookstore-service/src/main/resources/application.yaml

```diff
@@ -1,5 +1,12 @@
 server:
-  port: 8080
+  port: 8443
```

Comment:
Let the server be HTTP itself, and we can manage the SSL on the gateway ( nginx ) layer. As we ain't verifying the certificate, this setup should work with SSL on nginx.